crates/dslab-iaas/src/core/monitoring.rs

```diff
@@ -1,3 +1,5 @@
+//! Physical machines actual load.
```

Comment:
Добавьте к struct Monitoring:

```
/// This component stores the information about current host states received from host managers and provides this 
/// information to other components such as scheduler. Just like in a real system, the information arrives to the 
/// monitoring with some delay, so it can be outdated.
```